### ISSUE TITLE
[feature] viu * # result in images next to each other to use available width?

### ISSUE THREAD

**Issue by gwpl** [atanunq/viu#131]
Currently `viu *` will display images one under another.

Could viu fill the space displaing images next to each other ?
(and for current behaviour that is "one image per line" use same flag as `ls` i.e. `-1` ? )

```
I1 I2 I3 I4
I5 I6 I7 I8
(...)
```

?

Current behavior:

![Example of viu *.png displaying imaged one under another despite having space](https://github.com/user-attachments/assets/dc8901fe-014a-42e7-857e-a3b661b49526)

**Comment by enblanco** [comment]
Yes, please, implement grid display!. Something in the way [timg does it:](https://github.com/hzeller/timg#grid-display) would be 🥇. 

I use viu as my default img CLI viewer and i am missing this all the time.

**Comment by decatile** [comment]
As far as I discovered application does not know how much size would be occupied, it happens deep inside of viuer, so the most we can do - pass to program fixed amount of images we want to fit in a row at most.
